__mocks__/@fortawesome/react-fontawesome.js

```diff
@@ -0,0 +1,8 @@
+/**
+ * Reference: https://github.com/FortAwesome/react-fontawesome/issues/154
+ */
+import React from 'react';
+
+export function FontAwesomeIcon(props) {
+  return <i className={`fa ${props.icon}`} />;
+}
```

Comment:
What happens if we don't mock this?

Reply:
If we didn't mock this, everything runs as expected passing all the tests, and a bunch of errors (`console.error node_modules/@fortawesome/react-fontawesome/index.js:325
      Could not find icon { prefix: 'fas', iconName: 'minus-circle' }`) get printed in the console.
Mocking this import does not affect the passing/failing tests.

Not mocking it would be similar to render the UI with empty buttons, giving an error in the console, but it has no impact on the functionality of the application.

Reply:
@adamjarling Should we remove the mock?

Reply:
So the mocking just makes the testing output cleaner?   Less console errors more or less?   If so, I'm in favor of keeping the mock just to keep testing output clean and least confusing as possible.

Reply:
Yes, with the mock, the console comes out clean when running the test suite. Ok, I will merge this as it is :+1: